### PR TITLE
[chore] Removed $ from command for contribution guidance

### PR DESCRIPTION
**Pull request by gyliu513**
The copy in markdown file always copy the $ as well when copying the command, I need to remove the $ manually each time.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A

# Does This PR Require a Core Repo Change?

- [ ] Yes. - Link to PR: 
- [x] No.

# Checklist:

See [contributing.md](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md) for styleguide, changelog guidelines, and more.

- [ ] Followed the style guidelines of this project
- [ ] Changelogs have been updated
- [ ] Unit tests have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -124,31 +124,31 @@ pull requests (PRs).
...
+git remote add fork https://github.com/YOUR_GITHUB_USERNAME/opentelemetry-python-contrib.git
 ```
 
 Run tests:
 
 ```sh
 # make sure you have all supported versions of Python installed
-$ pip install tox  # only first time.
-$ tox  # execute in the root of the repository
+pip install tox  # only first time.
```
Comment (emdneto) [inline CONTRIBUTING.md:141]: Should we remove the comments so the code block works for copy/paste without problems?

Comment (gyliu513) [reply CONTRIBUTING.md:141]: Done

**Review by emdneto — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -126,31 +126,39 @@ pull requests (PRs).
...
+make sure you have all supported versions of Python installed, install `tox` only for the first time:
 
 ```sh
-# make sure you have all supported versions of Python installed
-$ pip install tox  # only first time.
-$ tox  # execute in the root of the repository
+pip install tox
+```
+
+Run tests in the root of the repository:
```
Comment (emdneto) [inline CONTRIBUTING.md:145]:
```suggestion
Run tests in the root of the repository (this will run all tox environments and may take some time):
```

Comment (gyliu513) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -126,31 +126,39 @@ pull requests (PRs).
...
-# edit files
-$ git commit
-$ git push fork feature
+git checkout -b feature
+```
+
+After you edit the files, run the following:
+
+```sh
+git commit
```
Comment (xrmx) [inline CONTRIBUTING.md:160]: Missing git add while we are at it?

Comment (gyliu513) [reply CONTRIBUTING.md:166]: Done, thanks @xrmx

**Review by lzchen — APPROVED** [review]

**Review by codefromthecrypt — APPROVED** [review]
I tend to use '$' when the triple backticked block includes the response to the command, basically where it is the console equiv of a screen shot.

Agree on removing it for the reasons you mention when there's nothing in the block except what to copy/paste

**Review by xrmx — APPROVED** [review]

**Pull request merged**